### PR TITLE
zfs style syntax

### PR DESCRIPTION
**Pull request by moogle19**
I changed the syntax to match the zfs style. _(0.7.7 Goals: Change syntax of ZFS commands to match ZFS syntax.)_
I also changed the way of getting the dataset location and mountpoint to match the style in 'ioh-guest' to make it not depend on hardcoded mountpoints and fixed pools. _(Preparing for 0.7.9 Goals: Allow iohyve datasets to live wherever the user wants them)_

**Breaking changes:**
-  `iohyve getall <guestname>` no longer exist and is replaced by `iohyve get all [guestname]` (Note: guestname is now optional. If not given all properties of all guests are printed.)
- `iohyve get <name> <prop>` is now `iohyve get <prop> <name>`
- `iohyve set <name> <property=value> ...` is now `iohyve set <property=value> ... <name>`

**Because it is a bigger change, it would be nice if you could test it and look for bugs.**

### CODE REVIEW

File: lib/ioh-zfs [hunk 1]
```diff
@@ -2,44 +2,75 @@
...
 	fi
-	local pool="$(zfs list -H -t volume | grep iohyve/$name | cut -d '/' -f 1 | head -n1)"
-	shift 2
+
+	local var=0
 	for arg in "$@"; do
-		local prop="$(echo $arg | cut -d '=' -f1)"
-		local val="$(echo $arg | cut -d '=' -f2)"
+		# Increment loop counter
+		var=$(($var+1))
```
Comment (moogle19) [inline lib/ioh-zfs:24]: I do not like the loop counter, but couldn't come up with a better solution.
Any suggestions?

**Comment by pr1ntf** [comment]
I'll dig around and take a look. Great work so far!

**Comment by pr1ntf** [comment]
That iteration function looks good to me. 

General ports and shell wizard Lars had used that method in his massive PR.

**Comment by pr1ntf** [comment]
No movement or discussion in three weeks, closing PR. I'll be adding changes made in #170 soon, causing even more conflicts.

**Pull request closed without merging**